Sources/Core/API/Animation.swift

```diff
@@ -147,8 +147,14 @@ internal extension Animation {
             return Frame(texture: textures[index], contentRect: contentRect)
         case .spriteSheet(let sheet):
             let framesPerRow = sheet.frameCount / sheet.rowCount
-            let row = index / framesPerRow
-            let column = index - row * framesPerRow
+            let rowIndex = index / framesPerRow
+            let column = index - rowIndex * framesPerRow
+
+            #if os(macOS)
```

Comment:
Instead of introducing another variable (which IMO makes the code a bit hard to follow), how about moving this conditional code to when we assign the content rect's origin? So something like:

```swift
#if os(macOS)
contentRect.origin.y = Metric(sheet.rowCount - 1 - row) / Metric(sheet.rowCount)
#else
contentRect.origin.y = Metric(row) / Metric(sheet.rowCount)
#endif
```

Reply:
Looks good, I'll change it!